library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -67,7 +59,6 @@ public WebNavigator withFallback(NavigationFallback navigationFallback) {
      * Provides a {@link IntentCustomizer} to be used to customize the Chrome Custom Tabs by attacking directly to
      * {@link SimpleChromeCustomTabsIntentBuilder}
      *
-     * @param intentCustomizer
```

Comment:
Unfortunately I do not have enough information in order to document all the previously documented parameters in this library @juankysoriano